src/devices/Adg731/samples/README.md

```diff
@@ -0,0 +1,102 @@
+# ADG731 - Samples
+
+## Required hardware
+* [Analog Devices ADG731](https://www.analog.com/media/en/technical-documentation/data-sheets/ADG725_731.pdf)
+* Any supported IoT device with an SPI bus
+
+## Hardware used for testing
+* [Bio Balance Detector Mars-64](http://www.biobalancedetector.com/) (with 2x ADG731s)
+* [Raspberry Pi 3 Model B+](https://www.raspberrypi.org/products/raspberry-pi-3-model-b-plus/)
+* [Pmod HAT Adapter](https://reference.digilentinc.com/reference/add-ons/pmod-hat/reference-manual) (Raspberry Pi Pmod HAT)
+* [Extech MN47 multimeter](http://www.extech.com/products/MN47) (for verifying switch completion)
+
+## Test unit and signal input
+![Raspberry Pi 3 Model B+ + Digilent Pmod HAT + BBD Mars-64 front side](Raspberry_3Bp_PmodHAT_BBDMars64_front.jpg)
+
+![Raspberry Pi 3 Model B+ + Digilent Pmod HAT + BBD Mars-64 back side](Raspberry_3Bp_PmodHAT_BBDMars64_back.jpg)
+
+
+## Installation on Raspberry Pi
```

Comment:
you can go ahead and create separate PR for deploying to PI instructions which can be used globally but please let's not add them per sample.